lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -375,6 +379,66 @@ BlockArgument FModuleOp::getPortArgument(size_t portNumber) {
   return getBodyBlock()->getArgument(portNumber);
 }
 
+/// Inserts the given ports. The insertion indices are expected to be in order.
+/// Insertion occurs in-order, such that ports with the same insertion index
+/// appear in the module in the same order they appeared in the list.
+void FModuleOp::insertPorts(
```

Comment:
As a utility, probably could be broken out also.

Reply:
Done!